docs/how-to-guides/integrating-autoware/creating-vehicle-interface-package/.pages

```diff
@@ -1,3 +1,5 @@
 nav:
-  - Creating a vehicle interface for an Ackermann kinematic model: creating-a-vehicle-interface-for-an-ackermann-kinematic-model.md
+  - vehicle-interface.md
```

Comment:
Maybe it's better to have page title more obvious
```suggestion
  - Vehicle Interface Overview: vehicle-interface.md
```

Reply:
Done :+1: